cmd/operator-sdk/run/local.go

```diff
@@ -179,18 +162,86 @@ func setupOperatorEnv(kubeconfig, namespace string) error {
 	return nil
 }
 
-func (c runLocalArgs) buildLocal(outputBinName string) error {
+// getBuildRunLocalArgs will return the Args to buil the project based on the flags used
```

Comment:
This comment is a bit vague. So maybe remove it or rephrase it:
```suggestion
// getBuildRunLocalArgs returns go build args for -ldflags and -gcflags
```